composer/README.md

```diff
@@ -0,0 +1,74 @@
+# Set up composer
+
+Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you.
+You will need composer for any of Symfony based project.
```

Comment:
"...any of Symfony based project**s**" (plural) or "...any Symfony based project".

Reply:
Changed in a5194d5